crates/matrix-sdk-ui/src/timeline/day_dividers.rs

```diff
@@ -480,6 +481,20 @@ impl DayDividerAdjuster {
             Some(report)
         }
     }
+
+    /// Returns whether the two dates for the given timestamps are the same or not.
+    fn is_same_date_as(
```

Comment:
Can we think of a more appropriate name for this function?

The best I can come up with right now is `should_be_separated_by_date_divider` but I would welcome better suggestions.

Reply:
I don't think this is always used to `separate` things so how about something very similar like `is_same_date_divider_group_as`?

Reply:
Yep, sounds good.